packaging/darwin/macos-pkg-build-and-sign.sh

```diff
@@ -46,7 +46,7 @@ sign "${binDir}/crc"
 sign "${binDir}/crc-admin-helper-darwin"
 sign "${binDir}/vfkit"
 
-signAppBundle "${BASEDIR}/root/Applications/Red Hat OpenShift Local.app"
+signAppBundle "${binDir}"
```

Comment:
this can be removed along with the `signAppBundle` function, we don't have the tray `.app` anymore, only signing the `crc`, `vfkit` and `admin-helper` binaries is enough i think